torch/quantization/_equalize.py

```diff
@@ -0,0 +1,130 @@
+import torch
+import copy
+
+def max_over_ndim(input, axis_list, keepdim=False):
+    ''' Applies 'torch.max' over the given axises
+    '''
+    axis_list.sort(reverse=True)
+    reduced_input = input
```

Comment:
There is no need for this line, as this is not a copy, but just getting the reference. The min/max function is not inplace, so you should not worry about input being modified